lib/auction.rb

```diff
@@ -0,0 +1,30 @@
+class Auction
+    attr_reader :items
+    def initialize
+        @items = []
+    end
+
+    def add_item(item)
+        @items << item
+    end
+
+    def item_names
+        @items.map(&:name)
+    end
+
+    def unpopular_items
+        @items.select {|item| item.bids.empty?}
+    end
+
+    def potential_revenue
+        @items.sum {|item| item.current_high_bid}
+    end
+
+    def bidders
```

Comment:
Technically, we wanted this to return an array of names, not the whole attendee object!  But, good use of map.  One thing to keep in mind is that you should try to keep your syntax consistent; because you are using one-line enums above, you should use a one-line here as well!